.github/workflows/stable.yml

```diff
@@ -149,31 +149,34 @@ jobs:
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: ${{ env.TOOLCHAIN }}
+          toolchain: ${{ env.TOOLCHAIN }}-${{ matrix.target }}
           target: ${{ matrix.target }}
           override: true
 
-      - name: Set up MinGW
+      - name: Set up MinGW for 64 bit
         if: matrix.target == 'x86_64-pc-windows-gnu'
-        uses: egor-tensin/setup-mingw@v2
+        uses: bwoodsend/setup-winlibs-action@v1
+
+      - name: Set up MinGW for 32 bit
+        if: matrix.target == 'i686-pc-windows-gnu'
+        uses: bwoodsend/setup-winlibs-action@v1
         with:
-          platform: x64
-          cc: 1
+          architecture: i686
 
       - name: Install dependencies
         uses: crazy-max/ghaction-chocolatey@v1
         with:
           args: install nasm
 
       - name: Run cargo build
-        if: matrix.target == 'x86_64-pc-windows-gnu'
+        if: endsWith(matrix.target, '-gnu')
         uses: actions-rs/cargo@v1
         with:
           command: build
           args: --target=${{ matrix.target }} --verbose --all-targets --features=ffi,qlog
 
       - name: Run cargo test
-        if: matrix.target != 'x86_64-pc-windows-gnu'
+        if: endsWith(matrix.target, '-msvc')
```

Comment:
I disabled test for Windows target because it was flappy (crashing). With this PR is it same?

Reply:
Yes. Initially I thought that it would not crash at random.